pkgs/by-name/sq/sqlelf/package.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, python3Packages
+, fetchPypi
+
+# ls and cat binaries are referenced in tests
```

Comment:
```suggestion
```
we can just look in code how that is used

Reply:
I think the comment is helpful FWIW

Reply:
https://github.com/NixOS/nixpkgs/pull/278142#issuecomment-1873501290

> I would like you to refrain from issuing stylistic reviews and leave this to the outcome of https://github.com/NixOS/rfcs/pull/166.

(Unfortunately the RFC does not apply to Bash code, but whatever.)